src/components/NavBar/NavBar.tsx

```diff
@@ -19,25 +20,43 @@ const useStyles = makeStyles<Theme>(theme => ({
 
 const NavBar: React.FC<NavBarProps> = props => {
   const classes = useStyles();
+  const [isMenuVisible, updateMenuVisibility] = useState(false);
 
   const button: NavBarButton = {
     menu: () => <MenuIcon />,
     back: () => <ArrowBackIcon />,
   };
   const { type, onClick } = props;
```

Comment:
is `onClick` still required?

Reply:
Nope 
You can preview it here: https://github.com/BuildIt-Poland/buddy-app/blob/fcdabc4d49685abb1895ea68c03909f304e6454c/src/components/NavBar/types.ts#L5